### ISSUE TITLE
Please support dynamic display of sidebar.

### ISSUE THREAD

**Issue by BopSun** [YU000jp/logseq-page-tags-and-hierarchy#2]
I use the TOC Generator plug-in, which can dynamically display the directory structure in the current page by typing {{renderer :tocgen, *}} in the sidebar directory page.

It would be great if your plug-in can also achieve this function!

**Comment by YU000jp** [comment]
I accept it as a suggestion.

This plugin only modifies stylesheets(CSS).
There is no Logseq Plugin API to get Page-tags and Hierarchy for a page yet.

**Comment by YU000jp** [comment]
It turns out that Logseq has a macro to show the hierarchy. 
Example: `{{namespace PAGE}}`

**Comment by YU000jp** [comment]
Sorry, let me rephrase. You want a table of contents on the right side of the page, like the page tags and hierarchy, instead of inside the open right sidebar, right?
